kbase-extension/scss/partials/_narrative.scss

```diff
@@ -1,24 +1,4 @@
-@import url('//fonts.googleapis.com/css?family=Roboto');
-
-.kb-dropzone {
-  #upload-message {
-    font-family: $typeface-page-text;
-    font-weight: 700;
-  }
-
-  .dz-message {
-    margin: 2em 4.5em;
-    color: rgb(0 0 0);
-    font: normal 400 24px/28px $typeface-alt-page-text;
-    mix-blend-mode: normal;
-  }
-}
-
-.btn__text,
-.kb-dropzone .dz-message {
-  font-style: normal;
-  text-align: center;
-}
```

Comment:
not all that useful having these combined, so I split them out. `dropzone`-related stanzas have been moved to be near the other `.kb-dropzone` and `.dz-*` stanzas.